numpy/core/src/umath/loops.c.src

```diff
@@ -997,9 +997,10 @@ NPY_NO_EXPORT void
 /**end repeat**/
 
 /**begin repeat
- * #TYPE = UBYTE, USHORT, UINT, ULONG, ULONGLONG#
- * #type = npy_ubyte, npy_ushort, npy_uint, npy_ulong, npy_ulonglong#
- * #c    = u,u,u,ul,ull#
+ * #TYPE  = UBYTE, USHORT, UINT, ULONG, ULONGLONG#
+ * #TYPE2 = BYTE, SHORT, INT, LONG, LONGLONG#
```

Comment:
Might name it SIGNED_TYPE? just a thought.

Reply:
Thanks yeah, really did not know what to call it :)